libshvbroker/src/aclmanager.cpp

```diff
@@ -402,7 +396,7 @@ chainpack::RpcValue AclManager::accessGrantForShvPath(const std::string& user_na
 	// find first matching rule
 	if(shv_url.pathPart() == BROKER_CURRENT_CLIENT_SHV_PATH) {
 		// client has WR grant on currentClient node
-		return cp::Rpc::ROLE_WRITE;
+		return {AccessLevel::Write};
```

Comment:
```suggestion
		return AccessLevel::Write;
```
Or is there a reason for the braces?